### PR TITLE
Functional tests - Separate view order page on blocks

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Separate view order page on blocks (customer, messages, products, payment, tab list (status, documents, carriers, merchandise returns))
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no need
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27583)

### CODE REVIEW

File: tests/UI/pages/BO/orders/view/tabListBlock.js [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+   * Click on enter payment button and get amount value
+   * @param page {Page} Browser tab
+   * @param row {number} Row on table
+   * @returns {Promise<*>}
+   */
+  async clickOnEnterPaymentButton(page, row = 1) {
+    await this.waitForSelectorAndClick(page, this.enterPaymentButton(row));
+
+    return page.$eval(paymentBlock.paymentAmountInput, el => el.value);
+  }
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/orders/view/tabListBlock.js]: Maybe split this in 2 functions, so you don't call another block here

```js
// tabListBlock.js
async clickOnEnterPaymentButton(page, row = 1) {
  await this.waitForSelectorAndClick(page, this.enterPaymentButton(row));
}


// PaymentBlock.js
getPaymentAmountInputValue(page) {
   return page.$eval(this.paymentAmountInput, el => el.value);
}
```

And On the test 
```js
await viewOrderPageTabList.clickOnEnterPaymentButton(page);
const amountValue = await viewOrderPageTabList.getPaymentAmountInputValue(page);
await expect(amountValue).to.not.equal('');
```

File: tests/UI/campaigns/sanity/02_ordersBO/02_editOrder.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ const loginCommon = require('@commonTests/loginBO');
 
 // Import pages
 const dashboardPage = require('@pages/BO/dashboard');
-const orderPage = require('@pages/BO/orders/view');
+const orderPage = require('@pages/BO/orders/view/productsBlock');
```
Comment (boubkerbribri) [inline tests/UI/campaigns/sanity/02_ordersBO/02_editOrder.js]: Applies here and all the other tests. Maybe choosing a more specific name for the page : 
`productsBlock.js`:  `orderPage` => `orderPageProductsBlock`
`customerBlock.js`:  `orderPage` => `orderPageCustomerBlock`
...

**Review by boubkerbribri — COMMENTED** [review]
Nice improvement for the project, Some small comments added

File: tests/UI/pages/BO/orders/view/tabListBlock.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/orders/view/tabListBlock.js]: Done thx :+1:

File: tests/UI/campaigns/sanity/02_ordersBO/02_editOrder.js (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/sanity/02_ordersBO/02_editOrder.js]: Done :+1:

**Review by matthieu-rolland — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**